odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewStats/OverviewQualityTestStats/OverviewQualityTestStats.tsx

```diff
@@ -1,91 +1,192 @@
+              {datasetsList?.length || 0}
+            </Typography>
+            <Typography variant="body2" className={classes.statLabel}>
+              datasets
+            </Typography>
+          </Grid>
+          <Grid item>
+            {datasetsList?.map(dataset => (
+              <AppButton
+                key={dataset.id}
+                className={classes.link}
+                size="small"
+                color="tertiary"
+                onClick={() => {}}
+              >
+                <Link to={dataEntityDetailsPath(dataset.id)}>
+                  {dataset.internalName || dataset.externalName}
+                </Link>
+              </AppButton>
+            ))}
+          </Grid>
+        </Grid>
+        <Grid item container className={classes.suites}>
+          <Grid item container>
+            <Typography variant="body2" className={classes.statLabel}>
+              Suite
+            </Typography>
+          </Grid>
+          <Grid item>
+            {suiteUrl ? (
+              <AppButton
+                className={classes.link}
+                size="small"
+                color="tertiary"
+                onClick={() => {}}
+              >
+                <Link to={{ pathname: suiteUrl }} target="_blank">
+                  {suiteName || suiteUrl}
+                </Link>
+              </AppButton>
+            ) : null}
+          </Grid>
+        </Grid>
       </Grid>
-      <Grid
-        item
-        container
-        xs={12}
-        direction="column"
-        alignItems="flex-start"
-      >
-        {datasetsList?.map(dataset => (
-          <AppButton
-            key={dataset.id}
-            className={classes.refItem}
-            size="small"
-            color="tertiary"
-            onClick={() => {}}
-          >
-            <Link to={dataEntityDetailsPath(dataset.id)}>
-              {dataset.internalName || dataset.externalName}
-            </Link>
-          </AppButton>
-        ))}
-      </Grid>
-    </Grid>
-    <Grid item container xs={6} className={classes.statsItem}>
-      <Grid item container xs={12} alignItems="baseline">
-        <Typography variant="body1" className={classes.statLabel}>
-          Suite
-        </Typography>
+      <Grid item className={classes.overview}>
+        <Grid item container xs={12} alignItems="baseline">
+          <Typography variant="body2" className={classes.statLabel}>
+            Overview
+          </Typography>
+        </Grid>
+        <LabeledInfoItem
+          inline
+          label="Last start"
+          labelWidth={4}
+          classes={{
+            value: cx(
+              classes.latestRunStatus,
+              qualityTest && qualityTest.latestRun?.status
+            ),
+          }}
+        >
+          {qualityTest && qualityTest.latestRun?.status}
+        </LabeledInfoItem>
+        <LabeledInfoItem inline label="Date" labelWidth={4}>
+          {qualityTest &&
+            qualityTest.latestRun?.startTime &&
```

Comment:
I think we can use 
```suggestion
            qualityTest?.latestRun?.startTime &&
```
And remove previous line.
Same can be applied to the similar code below.